contentprovider.go

```diff
@@ -212,6 +212,13 @@ func (p *contentProvider) fillContentMatches(ms []*candidateMatch) []LineMatch {
 		}
 		finalMatch.Line = data[lineStart:lineEnd]
 
+		if numContextLines > 0 {
+			finalMatch.Before = getLines(
+				data, p.newlines(), num-numContextLines-2, num-2)
```

Comment:
@kisabaka Final suggestion before merging, make the inputs to getLines based on 1-based indexing. IE just do `low -= 2` and `high -= 2` at the top of getLines with the helpful comment you added about why -2. This would make the call site much easier to read and reason correct.

Reply:
Sounds good!